src/CommunityToolkit.Maui.UnitTests/ImageSources/GravatarImageSource/GravatarImageSourceTests.cs

```diff
@@ -0,0 +1,414 @@
+﻿namespace CommunityToolkit.Maui.UnitTests.ImageSources.GravatarImageSource;
+
+using CommunityToolkit.Maui.ImageSources;
+using Xunit;
+
+public class GravatarImageSourceTests : BaseHandlerTest
+{
+	readonly TimeSpan cacheValidity = new(1, 0, 0);
+	readonly bool cachingEnabled = false;
+	readonly string email = "dsiegel@avantipoint.com";
+	readonly DefaultImage image = DefaultImage.MonsterId;
+
+	[Fact]
+	public void ChangeEmail()
```

Comment:
Just to confirm this method verifies that the `Uri` property does not change despite the email being set, is that correct? If so I wonder if we could rename this and the next 2 tests just to make that clear, reading this again I had to look into the code to determine why. Perhaps something like

`ChangingEmailWithNoSizeDoesNotUpdateUri` although perhaps less wordy :)